help/destinations/catalog/advertising/acxiom-audience-distribution.md

```diff
@@ -0,0 +1,143 @@
+|Address Line 2      |Address 2 field of primary residence                                                                                             |N|6            |255         |
+|City                |City of primary residence                                                                                                        |Y|7           |255         |
+|State               |State abbreviation of primary residence                                                                                          |Y|8           |2           |
+|Zip Code            |Full zip code of primary residence                                                                                               |Y|9            |10          |
+|Email               |Primary email By default, this field is used as a deduplication key to make the records unique                                   |N|10         |255         |
+|Phone               |Phone number of individual (area code + number)<br> By default, this field is used as a deduplication key to make the records unique. |N|11          |10          |
+
+
+1. Under the **Source Field** column, in the red select source field boxes, enter the name of each of the attributes you want to map from your input file to the corresponding target field. If you do not know the attribute name, you can search for the attribute by selecting the **Select Arrow** icon and then selecting the source field from the directory on the **Select source field** screen.<br>
+![Mapping screen](../../assets/catalog/data-partner/acxiom/mapping_screen.png)
+2. Once all your fields are mapped, select **[!UICONTROL Next]**.
+
+If you are not using [!DNL Adobe’s] standard schema, see the [Query Service UI guide](../../../query-service/ui/overview.md) documentation for information on how to use the query service to populate the [!DNL Adobe] standard schema with your field names. 
+
+
+
+### Review {#review}
+Once you have completed all the steps above, you have an opportunity to review your destination connection status and audience details before activating (distributing) it. The audiences you selected will show up at the bottom in a list. Each audience will be a separate call to the [!DNL Acxiom Audience Distribution] API. 
+
+If you are happy with the results, select **Finish** to activate your destination. 
+
+![Review your audience](../../assets/catalog/data-partner/acxiom/review_audience.png)
+
+
+
+## Troubleshooting {#troubleshooting}
+If your destination representative is unable to locate the audience, contact [!DNL Adobe] support for assistance in troubleshooting the delivery. 
+
+You will need the following information to provide to [!DNL Adobe] support:
+- Segment name
+- Destination name
+- Date of distribution
+- File name
+
+
+## Next steps
+
+By following this tutorial, you have successfully distributed an audience to the selected destination. Next, contact your destination representative to begin setting up your campaign.

```

Comment:
By following this tutorial, you have successfully activated one or multiple audiences to the selected destination platforms. Next, contact your destination platform representative to begin setting up your campaigns.